recipes-azure-iot/iot-hub-device-update/iot-hub-device-update/swupdate_handler_v2_grub.sh

```diff
@@ -0,0 +1,757 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
add our copyright information as well?

Reply:
yes, i saw now that our old v1 script had also the MS and the Conplement  copyright info.

@JanZachmann FYI.

Reply:
done